rfd/0123-tls-routing-behind-layer7-lb.md

```diff
@@ -299,15 +321,57 @@ for different clusters.
 UX is the same as when [TLS Routing is
 enabled](https://github.com/gravitational/teleport/blob/master/rfd/0039-sni-alpn-teleport-proxy-routing.md#local-teleport-proxy).
 
+#### 5 - Client source IPs
+
+A new setting is added to the Proxy service for web APIs to take client IPs from the "X-Forwarded-For" headers:
+```
+proxy_service:
+  # Enables the Proxy service to take client source IPs from the
+  # "X-Forwarded-For" headers for web APIs recevied from layer 7 load balancers
+  # or reverse proxies.
+  # 
+  # IMPORTANT: please make sure the service is behind a layer 7 load balancer or
+  # reverse proxy that sets or appends client IPs in the "X-Forwarded-For"
+  # headers.
+  #
+  # default: false
+  use_x_forwarded_for: true
```

Comment:
@klizhentas @xinding33 could you take a look at this new flag from UX point of view? Thanks!